LogicalTypes.md

```diff
@@ -655,10 +655,12 @@ should always be determined by the following rules:
 1. If the repeated field is not a group, then its type is the element type and
    elements are required.
 2. If the repeated field is a group with multiple fields, then its type is the
-   element type and elements are required.
-3. If the repeated field is a group with one field and is named either `array`
-   or uses the `LIST`-annotated group's name with `_tuple` appended then the
-   repeated type is the element type and elements are required.
+   element type and elements are required. In this case, the element type is
+   a Struct type with multiple fields.
+3. If the repeated field is a group (without annotation) with one `required` or
```

Comment:
I might leave the entire block as it was previously and clarify that:
1. Without annotatation the type is a single struct/tuple with the element in it.
2. With a logical type annotation (I think only LIST and MAP apply here).

I also think we need to update the documentation above:
```
The outer-most level must be a group annotated with `LIST` that contains a
  single field named `list`. The repetition of this level must be either
  `optional` or `required` and determines whether the list is nullable.
```
To point to the fact that for backwards compatibility it can also be repeated.

Reply:
> With a logical type annotation (I think only LIST and MAP apply here).

After double check, I don't think MAP annotation will appear here because it always has three levels.

> To point to the fact that for backwards compatibility it can also be repeated.

The original statement is all about the formal three-level encoding. Adding `repeated` to it may further confuse readers. That's why I tried to clarify this in the backward compatibility section.